politeiawww/www.go

```diff
@@ -100,8 +107,7 @@ func (p *politeiawww) getSessionUUID(r *http.Request) (string, error) {
 
 	id, ok := session.Values["uuid"].(string)
 	if !ok {
-		// No uuid in session so return "" to indicate that.
-		return "", nil
+		return "", ErrSessionUUIDNotFound
```

Comment:
should this be ```return nil, ErrSessionUUIDNotFound```?

Reply:
Not as it's written right now.  We could change it to that, but it would require updating the code that expects the returned UUID to be a string.